docs/en/Configuration-Settings.md

```diff
@@ -29,11 +29,12 @@ distributed with Tachyon binaries. We do not recommend beginner users to edit th
 which Tachyon is running. The easiest way is to put the site properties file in directory
 `$TACHYON_HOME/conf`.
 
-All Tachyon configuration properties fall into one of the five categories:
+All Tachyon configuration properties fall into one of the six categories:
 [Common](#common-configuration) (shared by Master and Worker),
-[Master specific](#master-configuration), [Worker specific](#worker-configuration), and
-[User specific](#user-configuration), and [Cluster specific](#cluster-management) (used for running
-Tachyon with cluster managers like Mesos and YARN).
+[Master specific](#master-configuration), [Worker specific](#worker-configuration),
+[User specific](#user-configuration), [Cluster specific](#cluster-management) (used for running
+Tachyon with cluster managers like Mesos and YARN), and [Security specific]
+(#security-configuration) (shared by Master, Worker, and User).
```

Comment:
Please move **(#security-configuration)** the end of previous line, otherwise it wouldn't be a link.